.github/workflows/compfuzzci_no_forks.yaml

```diff
@@ -0,0 +1,91 @@
+name: CompFuzzCI
+on:
+  pull_request:
+    branches:
+      - master
+    types: [opened, closed, synchronize, reopened]
```

Comment:
What is the types list for?

Reply:
I want to trigger workflow on pull request getting closed as well, this does not happen by default, hence the list.
By default pull_request event trigger on pull request being opened, synced, reopened.